content/release-10-10-0/devices-10-10-0-bundle/10-10-0-8.md

```diff
@@ -243,7 +243,7 @@ MTM-37556</td>
 <tr>
 <td>
 OPC UA</td>
-<td > If measurements overflow the range of a Long while converting from Java BigDecimal we convert them now to Double values.</td>
+<td > If measurements overflow the range of a long while converting from Java BigDecimal we convert them now to double values.</td>
```

Comment:
Perhaps we should add "value" here, i.e. "a long value", to make it clearer

Reply:
Fixed